pkgs/tools/security/analyzeMFT/default.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+}:
+
+python3Packages.buildPythonApplication rec {
```

Comment:
To fix the CI, you should remove `rec` (but only if you don't use variables recursively, like `${version}` in the suggestion below)